src/Symfony/Component/Routing/Tests/RouteTest.php

```diff
@@ -264,22 +264,4 @@ public function testSerializeWhenCompiled()
         $this->assertEquals($route, $unserialized);
         $this->assertNotSame($route, $unserialized);
     }
-
-    /**
-     * Tests that the serialized representation of a route in one symfony version
-     * also works in later symfony versions, i.e. the unserialized route is in the
-     * same state as another, semantically equivalent, route.
-     */
-    public function testSerializedRepresentationKeepsWorking()
```

Comment:
this test should be updated, not removed

Reply:
I'm sorry I removed it without comment.

I don't see how this test makes sense... It checks that a Route that has been created and serialized with a former release/version of Symfony (which one?) can be unserialized and actually equals the Route, including the regexp, that the current version would generate.

This obviously fails as the regexp is changed in this PR. Fiddling with the serialized string to have it reflect the "new" regexp cannot be a solution either.

Would you just check that the `unserialize` call itself works?

Maybe @Tobion you could explain what you intended to assert with this test?